applications/machine_learning/README.rst

```diff
@@ -285,19 +285,19 @@ For example, the configuration files for the Thingy:53 are defined in the :file:
 
 The following configuration files can be defined for any supported board:
 
-* :file:`prj_build_type.conf` - Kconfig configuration file for a build type.
-  To support a given build type for the selected board, you must define the configuration file with a proper name.
+* :file:`prj_<suffix>.conf` - Kconfig configuration file for a :ref:`suffixed configuration <cmake_options>`.
```

Comment:
For some configurations (e.g. Thingy:53) app uses child images. Maybe we should consider delaying transition to `FILE_SUFFIX` here too?
CC: @KAGA164